src/com/goide/runconfig/GoRunner.java

```diff
@@ -32,6 +33,9 @@ public String getRunnerId() {
 
   @Override
   public boolean canRun(@NotNull String executorId, @NotNull RunProfile profile) {
+    if(DefaultDebugExecutor.EXECUTOR_ID.equals(executorId)) {
```

Comment:
what's the point? The method is always false for DefaultDebugExecutor.EXECUTOR_ID without your changes.